manifests/j/JanDeDobbeleer/OhMyPosh/3.175.1/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 3.175.1
+PackageLocale: en-US
+Publisher: Jan De Dobbeleer
+PackageName: Oh My Posh
+License: GPL
+ShortDescription: Prompt theme engine for any shell
+Tags:
+- Console
+- Command-Line
+- Shell
+- Command-Prompt
+- PowerShell
+- WSL
+- Developer-Tools
+- Utilities

```

Comment:
```suggestion
Tags:
- console
- command-line
- shell
- command-prompt
- powershell
- wsl
- developer-tools
- utilities
```

Reply:
If this is a convention it should fail automatic validation. If it's a suggestion it should not block a merge.

Reply:
It is neither of them. If you have read my comment, it's just a best practice.